templates/telco/index.html

```diff
@@ -8,235 +8,220 @@
+      </p>
+      <p>
+        With 5G, elastic computing in the cloud becomes available to mobile users at zero latency. However, mobile handset capabilities are restricted and will remain a bottleneck. Offloading compute and storage  to the cloud via 5G networks could be a way to circumvent these limitations.
+      </p>
+    </div>
+    <div class="col-6 u-align--center u-hide--small">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/d5de891a-Acceleration.svg",
+        alt="",
+        width="287",
+        height="160",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
 
-        <section>
-            <div class="p-strip--light u-no-padding--bottom">
-                <div class="row">
-                    <div class="col-6">
-                        <h2>What’s under the hood</h2>
-                        <h4>Anbox Cloud builds on a powerful stack of open-source software to deliver Android app streaming from the cloud.</h4>
-                    </div>
-                </div>
-            </div>
+<section class="p-strip">
+  <div class="row u-vertically-center">
+    <div class="row">
+      <div class="col-7">
+        <h2>Inventing value-added services for 5G</h2>
+      </div>
+      <p>
+        5G makes ultra-rich media content, games, AR/VR, and even AI, accessible within telco RANs. With unrestricted computing capabilities available on demand, telcos can create innovative subscriber experiences across various mobile form factors to offer differentiated services to their subscribers.
+      </p>
+      <p>
+        What does it take to realise this vision? Considering the scale, this first and foremost requires servers in the cloud with an outstanding cost to performance ratio, equipped with application accelerators like GPUs, FGPAs and ASICs. Furthermore, to minimise latency, mobile workloads should be deployed at the edge, closer to mobile subscribers. Finally, instant elasticity and scalability are required to accommodate subscriber mobility patterns.
+      </p>
+    </div>
+    <div class="col-5">
+      <div class="p-card">
+        <h4>
+          Key challenges:
+        </h4>
+        <hr class="u-sv1">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Access to elastic compute </li>
+          <li class="p-list__item is-ticked">Deployment agility into edge RANs</li>
+          <li class="p-list__item is-ticked">Multi-cloud software automation</li>
+        </ul>
+        <p>
+          Anbox Cloud is engineered to address these challenges. The commercially supported platform builds on open source software to empower innovators to leverage Android at scale in the cloud.
+        </p>
+      </div>
+    </div>
+  </div>
+  <div class="row u-align--center">
+    <div class="col-4 u-hide--small">
```

Comment:
Move this `u-hide--small` to the row as we want the entire row hidden on small screens.